Task 1/index.js

```diff
@@ -0,0 +1,48 @@
+/* 
+Дан масив об'єктів. Вивести масив телефонних номерів користувачів, у яких баланс більше 2000 доларів. 
+І знайти суму всіх балансів користувачів
+*/
+
+import users from './data/data.json' assert { type: 'json' };
+import { transformAndDeleteCharacters } from './helpers/helper.js';
+
+const root = document.querySelector('.root');
+
+const findBalanceMore2k = (value) => {
+	const generateFilterArr = value.filter((item) => {
+		if (transformAndDeleteCharacters(item.balance) >= 2000) {
+			return item;
+		}
+	});
+
+	return generateFilterArr;
+};
```

Comment:
this function should return array of user phones by balance more than 2000$. So.
1. Naming! `filterUserPhonesByBalance`. 
2. When you see hardcoded value in task (2000), you should understand that this should be a parameter. So you need to call this function like
```
filterUserPhonesByBalance(users, 2000);
```
3. your function returns array of users, not their phones.